runtime/lua/vim/_editor.lua

```diff
@@ -74,7 +74,6 @@ vim.log = {
 --- Examples:
 ---
 --- ```lua
----
```

Comment:
Ehh again: In some places new blank lines were added, and here they were removed. Were all these changes intentional?

Reply:
The ones in the source were intentional so there are no diffs in the vimdoc files.

I left the ones in iter.lua as they look intentional.